src/Type/Php/StrRepeatFunctionReturnTypeExtension.php

```diff
@@ -74,6 +75,10 @@ public function getTypeFromFunctionCall(
 			$accessoryTypes[] = new AccessoryLiteralStringType();
 		}
 
+		if ($inputType->isNumericString()->yes()) {
+			$accessoryTypes[] = new AccessoryNumericStringType();
```

Comment:
I'm afraid this can be a bit tricky. Repeating a numeric type multiple times can lead to a non-numeric type: https://3v4l.org/aRBdZ

Reply:
So we should probably do it only with a constant (literal) types with no decimal points.

Reply:
good catch. fixed, thanks.

Reply:
I think it should only be done for strings that only contain digits. Because of these numeric strings: https://3v4l.org/U4bM2

It should also be ensured that the multiplier is not 0.

Reply:
ohh my, its even different across php versions ;)

https://3v4l.org/m6et3